docs/source/configuration.md

```diff
@@ -6,34 +6,28 @@ Following fields are required (and can be customized) in the [`challenge_config.
 
-- **description**: Long description of the challenge (use a relative path for the HTML file, e.g. `challenge_details/description.html`)
+- **description**: Long description of the challenge (use a relative path for the HTML file, e.g. `templates/description.html`)
 
-- **evaluation_criteria**: Evaluation criteria and details of the challenge (use a relative path for the HTML file, e.g. `challenge_details/evaluation.html`)
+- **evaluation_details**: Evaluation details and details of the challenge (use a relative path for the HTML file, e.g. `templates/evaluation_details.html`)
 
-- **terms_and_conditions**: Terms and conditions of the challenge (use a relative path for the HTML file, e.g. `challenge_details/tnc.html`)
+- **terms_and_conditions**: Terms and conditions of the challenge (use a relative path for the HTML file, e.g. `templates/terms_and_conditions.html`)
 
-- **image**: Logo of the challenge (use a relative path for the logo in the zip configuration, e.g. `images/logo/challenge_logo.jpg`). **Note**: The image must be in jpg, jpeg or png format.
+- **image**: Logo of the challenge (use a relative path for the logo in the zip configuration, e.g. `./logo.jpg`). **Note**: The image must be in jpg, jpeg or png format.
 
-- **submission_guidelines**: Submission guidelines of the challenge (use a relative path for the HTML file, e.g. `challenge_details/submission_guidelines.html`)
+- **submission_guidelines**: Submission guidelines of the challenge (use a relative path for the HTML file, e.g. `templates/submission_guidelines.html`)
 
-- **evaluation_script**: Python script which will decide how to evaluate submissions in different phases (path of the evaluation script file or folder relative to this YAML file. For e.g. `evaluation_script/`)
+- **evaluation_script**: Python script which will decide how to evaluate submissions in different phases (path of the evaluation script is in root e.g. `./`)
 
 - **remote_evaluation**: True/False (specify whether evaluation will happen on a remote machine or not. Default is `False`)
 
 - **is_docker_based**: True/False (specify whether the challenge is docker based or not. Default is `False`)
 
-- **is_static_dataset_code_upload**: True/False (specify whether the challenge is static dataset code upload or not. Default is `False`)
-
 - **start_date**: Start DateTime of the challenge (Format: YYYY-MM-DD HH:MM:SS, e.g. 2017-07-07 10:10:10) in `UTC` time zone
 
 - **end_date**: End DateTime of the challenge (Format: YYYY-MM-DD HH:MM:SS, e.g. 2017-07-07 10:10:10) in `UTC` time zone
 
 - **published**: True/False (Boolean field that gives the flexibility to publish the challenge once approved by EvalAI admin. Default is `False`)
 
-- **allowed_email_domains**: A list of domains allowed to participate in the challenge. Leave blank if everyone is allowed to participate. (e.g. `["domain1.com", "domain2.org", "domain3.in"]` Participants with these email domains will only be allowed to participate.)
```

Comment:
Why were these removed?

Reply:
Seeing from the reference config , it seemed that these were removed, but it seems model still have these so i will revert these changes.